.github/workflows/workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: Run unit tests
+on: ["push", "pull_request"]
+
+jobs:
+  test:
+    name: Dub tests
+    strategy:
+      matrix:
+        os: [macOS-latest, ubuntu-latest, windows-latest]
+        dc: [dmd-latest, ldc-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: 'Install D compiler'
+      uses: dlang-community/setup-dlang@v1
+      with:
+        compiler: ${{ matrix.dc }}
+
+    - name: 'Build and test'
+      run: |
+        # Test multiple builds
+          dub test --build=debug
+          dub test --build=release
+          dub test --build=release-debug
+          dub test --build=release-inline
```

Comment:
If you want to speed-up builds you can add the build type to the matrix and use `dub test --build=${{ matrix.build_type }}`

Reply:
To add something to the matrix you can simply write:
```yml
build_type: [debug, release, release-debug, release-inline]
```

Reply:
Weird, this message says it was sent 1 min after the first, but I've only seen it now. Anyways, that's what I did.